app/src/main/java/com/academy/ramon/marvelcomicviewer/util/HeroAdapter.java

```diff
@@ -62,7 +64,7 @@ public void onBindViewHolder(HeroAdapter.ViewHolder viewHolder, int position){
         TextView textView = viewHolder.heroName;
```

Comment:
@Mathreyu you could create a bind (Hero hero) method in the holder class for setting the values in the view as a good practice.